MobilityData/mobility-feed-api
# feat: filter out deprecated feeds from search

**Summary:**

- Added the possibility of specifying more than one status in the search.
- The UI is filtering out `deprecated` feeds (status=active,inactive,development)

Closes: #612

**Expected behavior:** 

The search endpoint accepts a list of status(comma-separated values). Example, `v1/search?status=inactive,active`

**Testing tips:**

Local testing:
- Populate local DB using,
```
./scripts/docker-localdb-rebuild-data.sh  --populate-db
```
- Start local API,
```
./scripts/api-start.sh
```
- Execute multiple combinations of API calls using status values. Example,
```
curl --location 'https://localhost:8080/v1/search?status=inactive%2Cactive' \
--header 'Accept: application/json'
```

UI screenshot:
<img width="1704" alt="Screenshot 2024-07-29 at 2 06 11 PM" src="https://github.com/user-attachments/assets/450fc58b-2b8a-4f0b-9cbe-20a3762f5318">


Please make sure these boxes are checked before submitting your pull request - thanks!
